client/src/components/FavButton/FavButton.js

```diff
@@ -1,28 +1,45 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "@emotion/styled";
-import favoriteIcon from "../../assets/icons/favorite-icon.svg";
 import PropTypes from "prop-types";
 
 const Button = styled.button`
-  border: 1px solid var(--color-dark-border);
-  background: var(--color-transparent);
+  border: 0.1px solid var(--color-dark-border);
```

Comment:
Wow, I'm pretty sure, MQ screens won't be able to show this border. I would choose at least 0.5px or so.

Reply:
I wanted to get rid of it and I am still thinking about it!